src/server/system_services/account_server.js

```diff
@@ -671,9 +671,15 @@ function list_accounts(req) {
         throw new RpcError('UNAUTHORIZED', 'Must be system admin or operator');
     }
 
-    // for support account - list all accounts
     const accounts = system_store.data.accounts
+        // for support account - list all accounts
         .filter(account => is_support || !account.is_support)
+        // filter list
+        .filter(account => req.rpc_params.filter &&
```

Comment:
if `req.rpc_params.filter` is undefined you will be filtering out all accounts... which is probably not intended.

Reply:
fixed